cmd/trigger_ui/list.html

```diff
@@ -1,107 +1,87 @@
-                                        Start
-                                    </th>
-                                    <th
-                                        class="px-5 py-3 border-b-2 border-gray-200 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider">
-                                        Status
-                                    </th>
-                                </tr>
-                            </thead>
 
-                            
-                            
-                            <tbody class="divide-y">
-                                {{$WorkflowStatuses := .WorkflowStatuses}}
 
-                                {{range $index, $element := .WorkflowList}}
-                                <tr class="bg-white text-sm border-gray-200 ">
-                                    <td class="px-5 py-5">
-                                        <p class="text-gray-900 whitespace-no-wrap">
-                                             {{$element.VxID}}
-                                        </p>
-                                    </td>
+							<tbody class="divide-y">
+								{{range $index, $element := .WorkflowList}}
+								<tr class="bg-white text-sm border-gray-200 ">
+									<td class="px-5 py-5">
+										<p class="text-gray-900 whitespace-no-wrap">
+										{{$element.VxID}}
+										</p>
+									</td>
 
-                                    <td class="p-5">
-                                        <p class="text-gray-900 whitespace-no-wrap">
-                                            {{$element.Name}}
-                                        </p>
-                                    </td>
-                                    <td class="p-5">
-                                        <p class="text-gray-900 whitespace-no-wrap">
-                                            {{$element.WorkflowID}}
-                                        </p>
-                                    </td>
-                                    <td class="p-5">
-                                        <p class="text-gray-900 whitespace-no-wrap">
-                                            {{$element.Start}}
-                                        </p>
-                                    </td>
+									<td class="p-5">
+										<p class="text-gray-900 whitespace-no-wrap">
+										{{$element.Name}}
+										</p>
+									</td>
+									<td class="p-5">
+										<p class="text-gray-900 whitespace-no-wrap">
+										{{$element.WorkflowID}}
+										</p>
+									</td>
+									<td class="p-5">
+										<p class="text-gray-900 whitespace-no-wrap">
+										{{$element.Start}}
+										</p>
+									</td>
 
-                                    
-       
-                                    <td class="p-5">
+									<td class="p-5">
+										{{range $index, $status := .WorkflowStatuses}}
```

Comment:
```suggestion
										{{range $index, $status := $.WorkflowStatuses}}
```

Since we are inside a loop, ".WorkflowStatuses" would represent $element.WorkflowStatuses here.
Thats why I had the  {{$WorkflowStatuses := .WorkflowStatuses}}.

But apparently we can also do $.WorkflowStatuses since $ represents the data passed to the template.

Reply:
Woops, sorry. Didn't think about that one. Thanks, will revert